models/default.go

```diff
@@ -0,0 +1,9 @@
+package models
+
+import "time"
+
+type Default struct {
```

Comment:
Not sure about the naming. How about `Base` instead as it's more common to use a base and extend from it. Just like these can be a base class and child classes 💭